t/protocol.t

```diff
@@ -33,6 +33,7 @@ __DATA__
                 end
 
                 ngx.log(ngx.DEBUG, "Upgrade: ", ngx.var.http_upgrade)
+                ngx.log(ngx.DEBUG, "Worker ID: ", conn.info.id)
```

Comment:
Should we add a same test in `privileged.t` to verify if worker_id == -1 ?

Reply:
added